app/components/profile_picture/profile_picture.js

```diff
@@ -45,20 +49,42 @@ export default class ProfilePicture extends PureComponent {
     }
 
     render() {
-        const {theme} = this.props;
+        const {edit, imageUri, showStatus, theme} = this.props;
         const style = getStyleSheet(theme);
 
         let pictureUrl;
         if (this.props.user) {
             pictureUrl = Client4.getProfilePictureUrl(this.props.user.id, this.props.user.last_picture_update);
         }
 
-        const statusIcon = (
-            <UserStatus
-                size={this.props.statusSize}
-                status={this.props.status}
-            />
-        );
+        if (edit && imageUri) {
+            pictureUrl = imageUri;
+        }
+
+        let statusIcon;
+        let statusStyle;
+        if (edit) {
```

Comment:
I'm not sure how I feel about these sharing a single component since it seems like they have different usage, but I can't think of a convincing enough argument to change it now